exporters/failure/collector_jira.py

```diff
@@ -279,4 +281,7 @@ def get_app_name(self, issue: Issue) -> str:
             matcher = f"{self.app_label}="
             if label.startswith(matcher):
                 return label.replace(matcher, "")
-        return "unknown"
+        if self.app_name is None:
+            # TODO set this behavior to all failure exporters?
```

Comment:
Should we add this to all failure exporters?

Reply:
I don't think so, maybe when there is a request. Also the sdp queries may require some modification as some of them do have "unknown" filter.